### PR TITLE
Clean up server features

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

**Review by marcus0x62 — APPROVED** [review]
LGTM!

File: bin/Cargo.toml [hunk 1]
```diff
@@ -32,57 +32,24 @@ default = ["sqlite", "resolver", "native-certs", "ascii-art"]
...
-    "dnssec",
-    "hickory-client/dnssec-ring",
-    "hickory-proto/dnssec-ring",
-    "hickory-server/dnssec-ring",
-]
+dnssec-openssl = ["dnssec", "hickory-server/dnssec-openssl"]
+dnssec-ring = ["dnssec", "hickory-server/dnssec-ring"]
 dnssec = []
 recursor = ["hickory-server/recursor"]
 # Recursive Resolution is Experimental!
```
Comment (AaronKutch) [inline bin/Cargo.toml:39]: Is this comment meant to go on the `recursor` feature?

Comment (djc) [reply bin/Cargo.toml:39]: Yes, it is (pre-existing issue). Want to send a PR?

Comment (AaronKutch) [reply bin/Cargo.toml:39]: It should go on this PR, otherwise this looks good

**Pull request merged**